test/test_env_utils.py

```diff
@@ -66,6 +66,22 @@ def test_prod_bucket_env():
     assert prod_bucket_env('fourfront-cgapwolf') is None
 
 
+def test_get_prd_or_stg_env():
+    assert get_prd_or_stg_env('data') in ['fourfront-green', 'fourfront-blue']
+    assert get_prd_or_stg_env('staging') in ['fourfront-green', 'fourfront-blue']
```

Comment:
This should probably mock a `boto3` client that implements `describe_environments` in a way that allows these to be tested more precisely and then test both states of the `CNAME` so that we know that this is doing the predicted thing in both cases; that is, when the `CNAME` is set one way, one of these should return blue and the other green.

Even without doing that, I think it would be an improvement to do (untested):
```
fourfront_prod_envs = {'fourfront-green', 'fourfront-blue'}
prd = get_prd_or_stg_env('data')
assert prd in fourfront_prod_envs
assert get_prd_or_stg_env('staging') == (fourfront-prod_envs - {prd}).pop()
```

Reply:
This is no longer relevant because you took out the function it applied to. But if you're curious about the mock stuff I was referring to, I was thinking of something similar to what I did in the mocks on [the added test in the last commit](https://github.com/4dn-dcic/utils/pull/94/commits/60618005eb68cb595b176aa602e462ec2706c914).